company-profiles/ebury.md

```diff
@@ -0,0 +1,29 @@
+# Ebury
+
+## Company blurb
+
+Ebury is a Global FinTech: we apply new technologies to enhance and automate financial services and processes. This allows small and medium-sized businesses to trade and transact internationally by eliminating boundaries related to more traditional procedures. 
+
+Founded in 2009, we are now positioned among the fastest-growing companies in the sector! Headquartered in London, we have more than 1000 staff covering over 50  nationalities (and counting!) working across more than 27 offices worldwide and serving more than 45,000 clients every day. 
+
+## Company size
+600-700 employees
+
+## Remote status
+At Ebury, we understand the key role that technology plays in supporting the growth and efficiency of financial services. This is why we invest so heavily in the latest tools, languages, and platforms, as well as promoting iterative development methods that allow us to work in a rapid evolution environment, creating continuous value.
+In the summer of 2020, our increasingly international technology team became remote-first, which means that working remotely is the default option for all the employees. In our remote onboarding process, Technology new employees have a ‘buddy’ in the team to accompany them throughout their first weeks. 
+
+## Region
+USA
```

Comment:
in the readme it shows "Worldwide" - which is correct @parthrc ?

Reply:
Their corporate offices are in USA but they operate worldwide, should I change the region here to worlwide?

Reply:
Yes, the region is for where the company operates/ hires employees.